flavorite_OAS_v1.yaml

```diff
@@ -0,0 +1,643 @@
+openapi: 3.0.3
+info:
+  title: Flavorite - OpenAPI 3.0
+  description: >-
+    This is a Flavorite Server based on the OpenAPI 3.0 specification. 
+  version: 1.0.0
+servers:
+  - url: https://localhost:8000/api/v1
```

Comment:
Please make sure the service port and API path (/api/v1)

Reply:
I was going to ask about this. I know the port should be changed from 8000 to 8080 to reflect the docker and readme info. However, I am unable to figure out how to set the base path to include /api/v1.

Reply:
https://fastapi.tiangolo.com/advanced/behind-a-proxy/

Reply:
Please see [relevant commit](https://github.com/flavorite/api-service/pull/1/commits/059e705759119b7d3b52271c2aee3144bf1a8a4f). Adds /api/v1 prefix to FastAPI app according to the owner's response to [this issue](https://github.com/flavorite/api-service/pull/1/commits/059e705759119b7d3b52271c2aee3144bf1a8a4f). Unsure if the .yaml files are reflective of this change.